velox/docs/monthly-updates/september-2023.rst

```diff
@@ -0,0 +1,93 @@
+* Add support for HUGEINT in function signatures :pr:`6799`
+* Add support for prioritizing productive sources in Exchange :pr:`6459`
+* Add support for UNKNOWN types to VectorHasher::hash :pr:`6435`
+* Add support for writing sorted buckted hive table :pr:`6142`
+* Extend Type with isOrderable() and isComparable() :pr:`6770`
+
+
+Presto Functions
+================
+
+* Add bitwise_xor_agg aggregate function :pr:`6705`
+* Add CAST TIMESTAMP WITH TIME ZONE to and from TIMESTAMP :pr:`6529`
+* Add geometric_mean aggregate function :pr:`6678`
+* Add overlapped string support in :func:`strpos` function. :pr:`6410`
+* Add reduce_agg aggregate function :pr:`6482`
+* Add :func:`to_ieee754_64` Function :pr:`6445`
+* Add support for all input types to IN predicate :pr:`6513`
+* Add support for complex types to compare argument of :func:`min_by` and :func:`max_by` :pr:`6605`
+* Add support for CAST DECIMAL to VARCHAR :pr:`6210`
+* Add support for UNKNOWN in :func:`map_from_entries` function :pr:`6353`
+* Add support for UNKNOWN type in :func:`arbitrary` aggregate function :pr:`6557`
+* Add support for VARBINARY input to :func:`length` function :pr:`6795`
+* Fix :func:`lead` and :func:`lag` window function for int64 offset :pr:`6463`
+
+
+Spark Functions
+===============
+
+* Add comparison Spark functions :pr:`5569`
+* Add support for all patterns in CAST VARCHAR to DATE :pr:`5844`
+* Add support for :spark:func:`rand` function with seed specified :pr:`6112`
+
+
+Hive Connector
+==============
+
+* Add native Parquet writer :pr:`6586`
+* Add S3FileSystem Write support :pr:`6021`
+* Fix Parquet requestType to align with scan table schema :pr:`6404`
+
+
+Performance and Correctness
+===========================
+
+* Add fast path of FlatMap column reader for scalar types :pr:`6507`
+* Fix array_remove null handling to follow Presto's behavior :pr:`6424`
+* Fix CPU stats tracking by attributing LazyVector loading time to TableScan :pr:`6558`
+* Fix handling null string in json_extract() to follow Presto's behavior :pr:`6439`
+* Fix min/max/min_by/max_by Presto aggregates when input contains nested nulls :pr:`6723`
+* Fix set_union Presto aggregate to follow Presto's behavior :pr:`6800`
+* Fix tracking of the number of processed rows in Window operator :pr:`6584`
+* Fix url_extract_path Presto function :pr:`6657`
+* Optimize exception handling in SIMDJsonExtractor parse and extract :pr:`6776`
+* Optimize large chunk memory allocation in stream arena :pr:`6547`
+* Optimize reduce_agg aggregation :pr:`6536`
+* Optimize spill performance by buffering spill data write :pr:`6509`
+* Optimize try_cast performance by reducing exception throws :pr:`5913`
+* Reduce peak memory usage by roughly half in HiveDataSource :pr:`6601`
+* Tune memory arbitration and spilling for Velox batch user case :pr:`6789`
+
+
+Build Systems
+=============
+
+* Install dependencies needed to support Azure Storage ABFS Connector :pr:`6418`
```

Comment:
Can we list these dependencies explicitly? CC: @kgpai

Reply:
This PR installs azure-sdk-for-cpp and its dependencies openssl and perl-IPC-Cmd.

Reply:
Thanks , azure/azure-sdk-for-cpp azure-storage-blobs_12.8.0 is the main dependency. We can say azure storage sdk.

Reply:
It also build sdk/core/azure-core though.